src/components/Exchange/PositionEditor.js

```diff
@@ -461,6 +463,35 @@ export default function PositionEditor(props) {
     [DEPOSIT]: t`Deposit`,
     [WITHDRAW]: t`Withdraw`,
   };
+  const ERROR_TOOLTIP_MSG = {
```

Comment:
1. Should we move this to ./constants too?
2. Can there be a case where there is an error code that is not in the config? Maybe we need to define a default message for that?

Reply:
1. Other messages can need access to token or position info. Like in the case of SwapBox here (https://github.com/gmx-io/gmx-interface/blob/8db708a53d976708dcd1caccba605927ebbc245a/src/components/Exchange/SwapBox.js#L1818). So I think we can let it be here for now. LMK you thoughts.
2. I have updated the condition which is responsible for rendering the Tooltip